server/infra/database/OrganizationRepositoryV2.ts

```diff
@@ -0,0 +1,104 @@
+import FilterOptions from 'interfaces/FilterOptions';
+import Organization from 'interfaces/Organization';
+import HttpError from 'utils/HttpError';
+import BaseRepository from './BaseRepository';
+import patch, { PATCH_TYPE } from './patch';
+import Session from './Session';
+
+export default class OrganizationRepository extends BaseRepository<Organization> {
```

Comment:
@dadiorchen stakeholder.stakeholder endpoint rather than entity. Which if you do that, then the getbyid no longer works the same way in v1.

Reply:
@dadiorchen stakeholder.stakeholder endpoint rather than entity. Which if you do that, then the getbyid no longer works the same way in v1.

Reply:
And I honestly believe that featured trees will have to be changed as well because the stakeholder.stakeholder uses uuid's instead of id's. I could be 100% wrong of course.

Reply:
I will get pictures of the difference to show what I mean

Reply:
/v2/organizations : 
![image](https://user-images.githubusercontent.com/63400531/216226788-07347e33-d0f4-4c2e-b0a2-76e6a3bc967e.png)

/query/orgnanizations:
![image](https://user-images.githubusercontent.com/63400531/216226910-12c41ef5-a3f5-4227-addb-5d20a23e3dc5.png)

/query/organizations/[id]
![image](https://user-images.githubusercontent.com/63400531/216227061-f94d6ccf-477a-42ed-be02-c1065dd736b1.png)

/v2/organizations/[id]
![image](https://user-images.githubusercontent.com/63400531/216227179-fe983116-376e-4607-98be-2c318d4cf382.png)

Reply:
Oh, you are considerting the stakeholder table, yea, you are right!